examples/react-es6/__tests__/CheckboxWithLabel-test.js

```diff
@@ -0,0 +1,26 @@
+jest.dontMock('../CheckboxWithLabel.js');
```

Comment:
Probably can leave out the `.js` here and the import statement (I don't know why it's there in the other React example).